### ISSUE TITLE
Changing new chat shortcut

### ISSUE THREAD

**Issue by adamshand** [iansinnott/prompta#16]
It would be great if the new chat shortcut could be changed.  

On macOS it's set to cmd-N which is also the shortcut for opening a new browser window for most browsers.

**Comment by iansinnott** [comment]
yeah, agreed. i've also run into this issue on the web version. I usually use cmd+k to open the menu then type "new" and hit enter. However, it would be good for the shortcut to work in the browser too.

Maybe ctrl+n. Open to suggestions though

**Comment by iansinnott** [comment]
added a quick fix in 3.1.0, not that the version number matters in the browser. the update is live
